services/ui_backend_service/data/db/tables/base.py

```diff
@@ -256,3 +256,29 @@ async def get_tags(self):
         except (Exception, psycopg2.DatabaseError) as error:
             self.db.logger.exception("Exception occured")
             return aiopg_exception_handling(error)
+
+    async def get_field_from(self, field: str, table: str, conditions: List[str] = [], values: List[str] = []):
```

Comment:
not sure the table is necessary as a parameter here? the function gets inherited by all table classes, so `self.table_name` should be available. Only change would be that you specifically call `.get_fields` on the table instance you want to perform it on

Reply:
Yep, Removed